src/content/en/updates/2018/03/webauthn-credential-management.md

```diff
@@ -0,0 +1,176 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: WebAuthn may break your website on Firefox and Edge if you are using the Credential Management API 
+
+{# wf_published_on: 2018-03-09 #}
+{# wf_updated_on: 2018-03-12 #}
+{# wf_featured_image: /web/updates/images/generic/security.png #}
+{# wf_tags: credentials,webauthn,sign-in #}
+{# wf_featured_snippet: WebAuthn may break your website on Firefox and Edge if you are using the Credential Management API #}
+{# wf_blink_components: Blink>SecurityFeature>CredentialManagement #}
+
+# WebAuthn may break your website on Firefox and Edge if you are using the Credential Management API {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+
+<div class="clearfix"></div>
+
+### TL;DR {: .hide-from-toc }
+[WebAuthn](https://www.w3.org/TR/webauthn/) helps increase security by bringing
+public-key credential based authentication to the Web, and is soon to be
+supported in Chrome, Firefox and Edge ([with the updated
+spec](https://docs.microsoft.com/en-us/microsoft-edge/dev-guide/device/web-authentication#differences-between-microsoft-edge-and-the-spec)).
+It adds a new kind of `Credential` object, which may break websites on Firefox
```

Comment:
s/which/which, however,/

Reply:
Also, upon discussing with Mike, let's not call names here:

It adds a new kind of Credential object, which may break websites that use the Credential Management API without feature-detecting the specific credential types they're interested in.